context/brand-context/springer/scss/40-base/typography.scss

```diff
@@ -12,24 +12,24 @@ h4,
 
 h1,
 .h1 {
```

Comment:
should these classes not be removed here?
e.g `.h1` not needed as we now have `.u-h1`